lib/omniauth/strategies/saml.rb

```diff
@@ -27,6 +27,7 @@ def self.inherited(subclass)
         first_name: ["first_name", "firstname", "firstName"],
         last_name: ["last_name", "lastname", "lastName"]
       }
+      option :relay_state
```

Comment:
Also, adding it as an option like this will mean that it gets passed to `ruby-saml` as a setting. It may make sense to `delete` it from the options before construction the `Settings` object.